README.md

```diff
@@ -136,6 +136,43 @@ rpc:
     batchDelay: 100
 ```
 
+#### RPC Block Receipts Enabled
+If this is `true`, will use `eth_getBlockReceipts` instead of `eth_getLogs`. Allows getting receipt data for transactions, but is not supported by every RPC. Default is `true`, but it will try to detect if the RPC supports `eth_getBlockReceipts` automatically.
```

Comment:
> but it will try to detect if the RPC supports `eth_getBlockReceipts` automatically

why "but"? it doesn't try to detect in case this config is set to `false`, right?

Reply:
```suggestion
If this is `true`, will use `eth_getBlockReceipts` instead of `eth_getLogs`. Allows getting receipt data for transactions, but is not supported by every RPC. Default is `true`, i.e. it will try to detect if the RPC supports `eth_getBlockReceipts` automatically.
```

Reply:
changed